frontend/src/config/Config.js

```diff
@@ -0,0 +1,5 @@
+const Config = {
+    serverUrl: "https://stormy-castle-43023.herokuapp.com"
```

Comment:
not sure if this api should be kept in a separate file as you have done, or move it into .env file so that it is kept private.

Reply:
Either way is good I suppose. Since URL does not contain any password or some sensitive info, therefore,  I kept it into config file .
Thanks